aries-site/src/data/wcag/components.json

```diff
@@ -12,6 +12,24 @@
       { "rule": "2.4.9", "status": "conditional" },
       { "rule": "3.2.1", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
+      { "rule": "1.4.11", "status": "passed" },
+      { "rule": "1.4.13", "status": "passed" },
+      { "rule": "1.4.3", "status": "passed" },
+      { "rule": "2.1.1", "status": "passed" },
+      { "rule": "2.1.2", "status": "passed" },
+      { "rule": "2.4.7", "status": "passed" },
```

Comment:
Kinda on the fence, but I might consider this as failing because the focus indicator is obscured (see screenshot, focus is on the first option but it isn't super clear because the focus indicator is cut off)
<img width="231" alt="Screenshot 2025-01-14 at 2 26 52 PM" src="https://github.com/user-attachments/assets/626d5420-1eaa-41da-9d6c-03dce3d8c07b" />